samples/word/99-preview-apis/manage-change-tracking.yaml

```
@@ -0,0 +1,155 @@
+          await Word.run(async (context) => {
+            context.document.body.clear();
+            context.document.body.insertParagraph(
+              "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
+              "Start"
+            );
+            context.document.body.paragraphs
+              .getLast()
+              .insertText(
+                "To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching Online cover page, header, and sidebar. Click Insert and then choose the Online elements you want from the different Online galleries.",
+                "Replace"
+              );
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows basic operations of the ChangeTracking feature.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+            <span class="ms-Button-label">Add sample text</span>
+          </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <h4>Get ChangeTracking mode</h4>
+            <button id="get-change-tracking-mode" class="ms-Button">
+                  <span class="ms-Button-label">Get ChangeTrackingMode</span>
+              </button>
+            <h4>Set ChangeTracking mode</h4>
+            <p>
+                <input type="radio" id="track-all-mode" name="mode" value="Track everyone's changes">
+                <label for="track-all-mode">Track everyone's changes</label><br>
+                <input type="radio" id="just-mine-mode" name="mode" value="Track only my changes">
+                <label for="just-mine-mode">Track only my changes</label><br>
+                <input type="radio" id="tracking-off-mode" name="mode" value="Turn change tracking off">
+                <label for="tracking-off-mode">Turn change tracking off</label><br>
+                <button id="set-change-tracking-mode" class="ms-Button">
+                    <span class="ms-Button-label">Set ChangeTrackingMode</span>
+                </button>
+            </p>
+            <h4>Get reviewed text</h4>
+            <p>
+                <label>First, select text in the document.</label>
```

Since there's only one written step, you can drop the "first". Also, if you imply the selection of tracked changes, it's a subtle hint to make edits.
```suggestion
                <label>Select text with tracked changes in the document.</label>
```